packages/console/src/pages/Cloud/pages/Congrats/index.tsx

```diff
@@ -0,0 +1,84 @@
+import classNames from 'classnames';
+import { useTranslation } from 'react-i18next';
+import { useNavigate } from 'react-router-dom';
+
+import AirPlay from '@/assets/images/air-play.svg';
+import Calendar from '@/assets/images/calendar.svg';
+import GetStarted from '@/assets/images/get-started.svg';
+import Button from '@/components/Button';
+import OverlayScrollbar from '@/components/OverlayScrollbar';
+import * as pageLayout from '@/pages/Cloud/layout.module.scss';
+import { buildUrl } from '@/utils/url';
+
+import ActionBar from '../../components/ActionBar';
+import { CloudPage } from '../../types';
+import { getCloudPagePathname } from '../../utils';
+import * as styles from './index.module.scss';
+
+const Congrats = () => {
+  const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const navigate = useNavigate();
+
+  const enterAdminConsole = () => {
+    navigate('/');
+  };
+
+  const handleBack = () => {
+    navigate(getCloudPagePathname(CloudPage.SignInExperience));
+  };
+
+  return (
+    <div className={pageLayout.page}>
+      <OverlayScrollbar className={pageLayout.contentContainer}>
+        <div className={classNames(pageLayout.content, styles.content)}>
+          <GetStarted />
+          <div className={styles.title}>{t('cloud.congrats.title')}</div>
+          <div className={styles.description}>{t('cloud.congrats.description')}</div>
+          <Button
+            type="primary"
+            size="large"
+            title="cloud.congrats.check_out_button"
+            icon={<AirPlay />}
```

Comment:
I'm curious why is this called `air play`?

Reply:
This icon is air play.
<img width="540" alt="image" src="https://user-images.githubusercontent.com/10806653/219315008-8d281a84-3d47-41be-bf11-8ad895d33c11.png">

Reply:
Will update it in another PR.